perses/utils/openeye.py

```diff
@@ -17,6 +19,38 @@
 _logger = logging.getLogger("utils.openeye")
 _logger.setLevel(logging.INFO)
 
+def system_generator_wrapper(oemols,
```

Comment:
Is it worth adding this extra level of abstraction?

Reply:
i think that this is ok, because it's basically just holding a bunch of defaults that we would like to use in one place. The use for this will go once we get further with the Experiments, but it's not causing any harm right now

Reply:
No problem, but should I change the defaults for `SystemGenerator` so we wouldn't need this?